tests/models/test_chgnet.py

```diff
@@ -0,0 +1,246 @@
+import traceback
+
+import pytest
+import torch
+from ase.atoms import Atoms
+
+import torch_sim as ts
+from tests.conftest import DEVICE
+from tests.models.conftest import make_validate_model_outputs_test
+
+
+try:
+    from chgnet.model.model import CHGNet
+
+    from torch_sim.models.chgnet import CHGNetModel
+except (ImportError, ValueError):
+    pytest.skip(
+        f"CHGNet not installed: {traceback.format_exc()}", allow_module_level=True
+    )
+
+
+DTYPE = torch.float32
+
+
+@pytest.fixture
+def ts_chgnet_model() -> CHGNetModel:
+    """Create a TorchSim CHGNet model for testing."""
+    return CHGNetModel(
+        device=DEVICE,
+        dtype=DTYPE,
+        compute_forces=True,
+        compute_stress=True,
+    )
+
+
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64])
+def test_chgnet_dtype_working(si_atoms: Atoms, dtype: torch.dtype) -> None:
+    """Test that CHGNet works with different dtypes."""
+    model = CHGNetModel(
+        device=DEVICE,
+        dtype=dtype,
+        compute_forces=True,
+    )
+
+    state = ts.io.atoms_to_state([si_atoms], DEVICE, dtype)
+    result = model.forward(state)
+
+    # Check that results have correct dtype
+    assert result["energy"].dtype == dtype
+    assert result["forces"].dtype == dtype
+    assert result["stress"].dtype == dtype
+
+
+def test_chgnet_batched_calculations() -> None:
+    """Test that CHGNet handles batched calculations correctly."""
+    from ase.build import bulk
```

Comment:
Many of the tests here are redundant with the tests in `torch_sim.models.tests.make_validate_model_outputs_test` and `torch_sim.models.tests.make_model_calculator_consistency_test`. I'd recommend taking a look at how some of the other models are tested ([like mattersim](https://github.com/TorchSim/torch-sim/blob/main/tests/models/test_mattersim.py)) and following that paradigm.